### PR TITLE
Deprecate Statement::getTimestamp()

### PR DESCRIPTION
**Pull request by Lctrs**
Let's not have 2 getters that does the same.

### CODE REVIEW

File: src/Statement.php [hunk 1]
```diff
@@ -266,10 +266,14 @@ public function getAuthority()
      * occurred.
      *
      * @return \DateTime The timestamp
+     *
+     * @deprecated since 1.1.1, to be removed in 3.0
```
Comment (xabbuh) [inline src/Statement.php]: please change this to `1.2`, I don't want to introduce new deprecations in a patch release

File: src/Statement.php [hunk 2]
```diff
@@ -266,10 +266,14 @@ public function getAuthority()
...
      */
     public function getTimestamp()
     {
-        return $this->created;
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.1.1 and will be removed in 3.0. Use "%s::getCreated()" method instead.', __METHOD__, __CLASS__), E_USER_DEPRECATED);
+
+        return $this->getCreated();
```
Comment (xabbuh) [inline src/Statement.php]: Let's not change this one. There's not really a need for an additional method call here.

File: src/Statement.php [hunk 3]
```diff
@@ -266,9 +266,13 @@ public function getAuthority()
      * occurred.
      *
      * @return \DateTime The timestamp
+     *
+     * @deprecated since 1.2, to be removed in 3.0
      */
     public function getTimestamp()
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.1.1 and will be removed in 3.0. Use "%s::getCreated()" method instead.', __METHOD__, __CLASS__), E_USER_DEPRECATED);
```
Comment (xabbuh) [inline src/Statement.php]: So we now just need to make the change here too. :)

Comment (Lctrs) [reply src/Statement.php]: Yeah I just saw it 🥇

**Comment by Lctrs** [comment]
@xabbuh I spotted another thing to be deprecated. Don't merge this now please.

**Comment by Lctrs** [comment]
@xabbuh Should be good now.

**Comment by xabbuh** [comment]
Thank you @Lctrs.

**Pull request closed without merging**